src/olympia/amo/sitemap.py

```diff
@@ -112,34 +116,46 @@ class AMOSitemap(Sitemap):
     # i18n = True  # TODO: support all localized urls
     changefreq = 'always'
     lastmod = datetime.datetime.now()
+    apps = None  # because some urls are app-less, we specify per item
 
     def items(self):
         return [
             # frontend pages
-            'home',
-            'pages.about',
-            'pages.review_guide',
-            'browse.extensions',
-            'browse.extensions.categories',
-            'browse.themes',  # TODO: when we add /android, .themes are /firefox only
-            'browse.themes.categories',
-            'browse.language-tools',  # TODO: when we add /android this is /firefox only
+            ('home', amo.FIREFOX),
+            ('home', amo.ANDROID),
+            ('pages.about', None),
+            ('pages.review_guide', None),
+            ('browse.extensions', amo.FIREFOX),
+            ('browse.extensions', amo.ANDROID),
+            ('browse.extensions.categories', amo.FIREFOX),
+            ('browse.extensions.categories', amo.ANDROID),
```

Comment:
Did we discuss excluding categories for Android, or is this something else?

Reply:
ah, this is the page that shows the _list_ of categories https://addons.mozilla.org/en-US/firefox/extensions/categories/ though I'm not sure we need that at all (or why it exists as a page).  

The addons in a particular category is handled in `CategoriesSitemap` (I thought that's what you were referring to in the issue)

Reply:
I'll drop this url for Firefox too.